rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -122,9 +122,8 @@ def lswitch_del(self, name):
             self.run("ls-del", args=params)
 
 
-
         def lswitch_list(self):
-            self.run("ls-list")
+            return self.run("ls-list | awk '{print $2}'")
```

Comment:
we should not change the behavior of lswitch_list. Parsing the information can be done in the caller, according to the caller's needs. For example, in this patch in ovn._list_lswitches() it anyway performances extra parsing, so this awk here is not very valuable while losing information of the original command output.

Reply:
It is debatable as this just gives the results what we need. Anyways we will have to parse lswitches data. So better do it at a calling time!

Reply:
> It is debatable as this just gives the results what we need.
The lswitch uuids could be useful for other callers, so I'd suggest to keep the original information. If in every caller we end up doing same processing to filter out the uuids and only need names, it makes sense to pre-precess then.